assets/js/base/context/providers/cart-checkout/shipping/index.tsx

```diff
@@ -191,15 +186,12 @@ export const ShippingDataProvider = ( { children } ) => {
 		currentErrorStatus.hasInvalidAddress,
 	] );
 
-	/**
-	 * @type {ShippingDataContext}
-	 */
 	const ShippingData = {
 		shippingErrorStatus: currentErrorStatus,
 		dispatchErrorStatus,
 		shippingErrorTypes: ERROR_TYPES,
 		...eventObservers,
-	};
+	} as ShippingDataContextType;
```

Comment:
You can remove `as` here, and instead type the variable, so

```ts
	const ShippingData: ShippingDataContextType = {
		shippingErrorStatus: currentErrorStatus,
		dispatchErrorStatus,
		shippingErrorTypes: ERROR_TYPES,
		...eventObservers,
	};
```

the `as` "forces" the variable to be of the type specified, even if it types do not overlap, for example:

```ts
	const ShippingData = {
		shippingErrorStatus: currentErrorStatus,
		dispatchErrorStatus,
		madeUpKey: true,
		shippingErrorTypes: ERROR_TYPES,
		...eventObservers,
	} as ShippingDataContextType;
``` 

does not show a type error!